docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -108,18 +100,18 @@ Block blob of length 505623: https://<accountname>.blob.core.windows.net/photos/
 
 To download blobs, first retrieve a blob reference and then call the `DownloadTo` method. The following example uses the `DownloadTo` method to transfer the blob contents to a stream object that you can then persist to a local file.
 
-[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L63-L69)]
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L58-L64)]
```

Comment:
You *may* wish to *consider* removing the use of `__SOURCE_DIRECTORY__` in the middle of the code and replacing it with e.g.

```fsharp
System.Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
```

at the top of the file (or wherever). That way, you can simply do

```fsharp
File.OpenWrite("path/download.txt")
```

etc.


However, this will mean explaining it up front which may not be ideal.